poc_mobile_verifier/src/verifier.rs

```diff
@@ -46,8 +46,11 @@ impl VerifierDaemon {
                 || epoch_since_last_reward_duration >= reward_period
             {
                 let epoch_duration = epoch_since_last_verify_duration.min(verification_period);
+                let last_rewarded_end_time = Utc.timestamp(*self.last_verified_end_time.value(), 0);
                 let last_verified_end_time = Utc.timestamp(*self.last_verified_end_time.value(), 0);
-                let epoch = last_verified_end_time..(last_verified_end_time + epoch_duration);
+                let epoch = last_verified_end_time
+                    ..(last_verified_end_time + epoch_duration)
+                        .min(last_rewarded_end_time + Duration::hours(24));
```

Comment:
This would limit the epoch to 24 hours right? Which would exclude actual being able to do a catch up across more than that if we're done for a length of time? Why is it being constrained?

Reply:
Or is the `Duration::zero()` below telling me that you'd issue multiple 24 hour intervals instead of one "big one"

Reply:
We'd issue multiple 24 hour periods (or any period actually, restricting to 24 was a mistake). The reason for this is that if we make the epoch based on `Utc::now()`, we continually drift longer than 24 hours.